src/iptax/ai/review.py

```
@@ -0,0 +1,596 @@
+            old_row = self._row_widgets[old_index]
+            old_row.remove_class("selected")
+        
+        # Update new row - add selected class
+        if 0 <= new_index < len(self._row_widgets):
+            new_row = self._row_widgets[new_index]
+            new_row.add_class("selected")
+            
+            # Check if new row is visible, scroll if needed
+            self._ensure_row_visible(new_row, new_index)
+
+    def _ensure_row_visible(self, row: Static, index: int) -> None:
```

Ruff (0.14.7)

Unused method argument: `row`

(ARG002)